### ISSUE TITLE
Template strings with leading regexp literal expression

### ISSUE THREAD

**Issue by kentor** [6to5/6to5#512]
```javascript
`${/\d/.exec("1")[0]}`
```

Gives
```
SyntaxError: stdin: Unexpected token (1:3)
> 1  | `${/\d/.exec("1")[0]}`
     |   ^
  2  |
```

Expected
```javascript
"" + /\d/.exec("1")[0];
```

or something similar.

**Comment by sebmck** [comment]
Can you report this upstream to [acorn](https://github.com/marijnh/acorn) please?

**Comment by kentor** [comment]
Upstream is patched

**Comment by sebmck** [comment]
@kentor So 6to5 uses `acorn-6to5` which is a fork of `acorn-jsx` which is a fork of `acorn`. I'm currently waiting on some changes to land in `acorn-jsx` (see [https://github.com/RReverser/acorn-jsx/issues/13#issuecomment-70367178](RReverser/acorn-jsx#13)) and once given the go ahead by @RReverser I'll merge in the latest.

**Comment by sebmck** [comment]
Fixed in the 3.0.0 branch
